website/pages/en/developing/unit-testing-framework.mdx

```diff
@@ -559,14 +566,14 @@ export function createNewGravatarEvent(
     mockEvent.transaction,
     mockEvent.parameters
   )
-  newGravatarEvent.parameters = new Array()
-  let idParam = new ethereum.EventParam('id', ethereum.Value.fromI32(id))
-  let addressParam = new ethereum.EventParam(
+  newGravatarEvent.parameters = []
```

Comment:
replaced `new Array()` with `[]`

Reply:
This is AssemblyScript and I think we need to declare it like that https://www.assemblyscript.org/stdlib/array.html#array